components/spicedb/codegen/codegen.go

```diff
@@ -158,20 +162,38 @@ func generateFluentAPI(def *corev1.NamespaceDefinition) string {
 
 		objects := ``
 		for _, t := range rel.TypeInformation.AllowedDirectRelations {
-			objects += `
+			if t.GetPublicWildcard() != nil {
```

Comment:
Nice, need the same in the "workpsace sharing" PR, and did not yet look into it, yet. :orange_heart: